lib/src/models.dart

```diff
@@ -172,4 +162,52 @@ class Move {
 
   @override
   int get hashCode => hashValues(from, to, promotion);
+
+  static Move? make(String? uci) {
```

Comment:
You should use a [named constructor](https://dart.dev/guides/language/language-tour#named-constructors) instead of a static method, and name it `Move.fromUci(String uci)` so it's consistent with `Coord.fromSquareId`. `uci` param should not be optional